package/yast2-installation-control.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Oct 10 10:29:07 UTC 2017 - jreidinger@suse.com
+
+- add self_update_id to allow define id for self update scc query
+  (bsc#1057538)
```

Comment:
Is this the correct ID? I don't see how https://bugzilla.suse.com/show_bug.cgi?id=1057538 is related.

Reply:
good catch, I copy it from wrong tab, correct one is https://bugzilla.suse.com/show_bug.cgi?id=1055556